packages/teams-js/test/private/conversations.spec.ts

```diff
@@ -32,6 +33,19 @@ describe('conversations', () => {
       );
     });
 
+    it('conversation capability is not supported', async () => {
```

Comment:
Should this say something like `throw not supported on platform error if conversation capability is not supported`?